src/main/java/sparksoniq/jsoniq/item/FunctionItem.java

```diff
@@ -59,7 +60,7 @@ protected FunctionItem() {
     public FunctionItem(
             FunctionIdentifier identifier,
             List<String> parameterNames,
-            List<SequenceType> signature,
+            FunctionSignature signature,
```

Comment:
@CanBerker do you approve this minor refactoring (a proper class for function signatures, which will also be convenient at many other places).

Reply:
This is perfectly fine by me. I was actually anticipating a class as a signature but did not want to implement that myself as it would most likely be subject to a similar refactoring when Andrea would work on it.

Reply:
Thanks @CanBerker 

@AndreaRinaldi1 then it's good to go.